doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-            \scalebox{0.255}[0.27]{\includegraphics{img/exp-smoothings.pdf}}
-            \caption{Exponential smoothing models on \emph{austourists}.}
-            \label{img:exp-smoothings}
-        \end{center}
-    \end{figure}
-
-    \begin{figure}[H]
-        \begin{center}
-            \scalebox{0.255}[0.27]{\includegraphics{img/arima-sarima.pdf}}
-            \caption{Arima models on \emph{austourists}.}
-            \label{img:arimas}
-        \end{center}
-    \end{figure}
-
-    ARIMA models are showed in \ref{img:arimas}. Non seasonal model is already capable of capturing some seasonal
-    patterns but it is more random whereas seasonal patterns should be of fixed period. Therefore seasonal ARIMA with
-    dedicated model for each period is more appropriate.
-
-    \begin{table}[h]
-        \begin{center}
-            \begin{tabular}{l|r|r|c}
-                    \textbf{Model} & \textbf{MSE} & \textbf{MAE} & \textbf{AIC} \\ \hline \hline
-                    Na\"{i}ve & 104.74 & 8.64 & \\
-                    Drift & 104.60 & 8.47 & \\
-                    Average & 79.91 & 7.14 & \\ \hline
-                    Simple exp. smoothing & 53.54 & 5.91 & 380.88 \\
-                    Double exp. smoothing & 44.41 & 5.28 & 375.90 \\
-                    Triple exp. smoothing & 5.34 & 1.71 & 282.73 \\ \hline
-                    Linear regression & 41.85 & 5.10 & 321.46 \\ \hline
-                    ARIMA(2, 1, 2) & 30.88 & 4.35 & 310.6 \\
-                    SARIMA(1, 0, 0)(1, 1, 0)[4] & 4.79 & 1.63 & 206.95 \\
-            \end{tabular}
-            \caption{Statistics of models.}
-            \label{tab:models-stat}
-        \end{center}
-    \end{table}
-
-    Statistics produced by each model are listed in table \ref{tab:models-stat}. Models with the lowest one
-    step ahead prediction error are seasonal ARIMA and triple exponential smoothing which is correct for this highly
-    seasonal time series.
-
-    As it was discussed in section \ref{sec:model-quality} for models comparison should be used AIC criterion. From the
-    table \ref{tab:models-stat} it can be seen that AIC and MSE are related, however MSE of double exponential smoothing
-    is $8.3$ times higher than MSE of triple exponential smoothing. Whereas AIC of double exponential smoothing is
-    only $1.3$ times higher. From this example it is obvious how AIC discriminates models with more parameters.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Existing Solutions}
-Before going directly to the design and implementation let's briefly look at existing monitoring solutions and libraries
-for time series modelling.
-
-In the next section are described not only direct competitors of Hawkular but also
-generally available software of this kind. Because Hawkular is still only an upstream project and not deliverable
-product for the customers thereby as its alternative is considered Jboss Operations Network (JON).
```

Comment:
...customers, thereby Jboss Operations Network (JON) is considered as its alternative.